packages/paste-core/components/button/src/index.tsx

```diff
@@ -116,4 +116,4 @@ Button.defaultProps = {
 
 Button.displayName = 'Button';
 
-export {Button};
+export {Button, ButtonProps};
```

Comment:
I don't think  this is valid, should do what we do with AnchorProps and export it separately.  This creates an object of a JS value and a TS types value which should error.

Reply:
Well it does seem to build fine but I haven’t checked the output.

Could simply `export * from ‘./types’` to be safe?